packages/containers/src/ComponentForm/ComponentForm.sagas.js

```diff
@@ -75,13 +75,18 @@ function* onFormSubmit(componentId, submitURL, action) {
 	}
 	const { response, data } = yield call(cmf.sagas.http.post, submitURL, action.properties);
 	if (!response.ok) {
-		return;
+		yield put({
+			type: Component.ON_SUBMIT_FAILED,
+			data,
+			componentId,
```

Comment:
please add element from the response in it so we can build any effects on it

Reply:
I add the response :) good call